specification/vmware/Microsoft.AVS.Management/scriptExecutions.tsp

```diff
@@ -146,9 +146,8 @@ model ScriptExecutionProperties {
 
   #suppress "@azure-tools/typespec-azure-core/bad-record-type" "It is unknown."
   #suppress "@azure-tools/typespec-azure-resource-manager/arm-no-record" "It is unknown."
-  #suppress "@azure-tools/typespec-azure-resource-manager/no-empty-model" "Existing use case"
   @doc("User-defined dictionary.")
-  namedOutputs?: Record<{}>;
+  namedOutputs?: Record<unknown>;
```

Comment:
I'll ask our scrpiting team for some `namedOutputs` examples to verify this change.

Reply:
I see in `clients.tsp`, the java & go SDKs are already overriding this.

```
@@alternateType(ScriptExecutionProperties.namedOutputs,
  Record<unknown>,
  "java,go"
);
```

Reply:
Would it be simpler to add `csharp` to that list?

Reply:
csharp's generated code was using `unknown` for this property.
Therefore for csharp, we need to use `unknown` as the alternative type.

Reply:
It is now set to `Record<unknown` for compatibility.